TypeCobol.Test/Parser/Programs/Cobol85/IncorrectSetUpTo.rdz.cbl

```diff
@@ -0,0 +1,13 @@
+﻿       IDENTIFICATION DIVISION.
+       PROGRAM-ID. RB1BCTLB.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+       01 W-TAB.
+          05 W-TAB2 OCCURS 50 INDEXED BY I-DTL.
+            10 W-TAB-ELT                  PIC X(07).
+       01 I-TAB-MAX                           PIC S9(02) COMP-5.
+      
+       PROCEDURE DIVISION.
+           SET I-TAB-MAX                 UP TO 1
```

Comment:
Can this invalid syntax test be merged in an existing test for SET intructions instead of adding a new file ?

Reply:
@fm-117 
Yes, good idea. @delevoye can you add this test to  TypeCobol/TypeCobol.Test/Parser/CodeElements/SETForIndexes.cbl ?

Reply:
@delevoye 
Can you add the following test also ?
```cob
SET I-TAB-MAX              DOWN BY 1
```

Reply:
This test already existed, so I added the ligne with the erroneous `DOWN TO` to the test instead.